nzedb/IRCScraper.php

```diff
@@ -135,11 +136,6 @@ public function __construct(&$silent = false, &$debug = false)
 		$this->log = new Logger();
 	}
 
-	public function __destruct()
-	{
-		parent::__destruct();
-	}
-
```

Comment:
Not sure about this. Despite what PHP's own documentation says on constructors and destructors there are known to be some edge cases when one is not implemented explicitly. I'd prefer to leave these in for safety's sake until those matters have been cleared up conclusively.